app/src/utils/data/contributions.ts

```diff
@@ -56,7 +56,7 @@ export async function getContributions(
 
               const limit = 100;
 
-              const fetchUntilAll = async (SUBGRAPH_URL: string, before = [], skip = 0): Promise<any[]> => {
+              const fetchUntilAll = async (SUBGRAPH_URL: string, before: any[] = [], skip = 0): Promise<any[]> => {
```

Comment:
Ideally we should have stronger typing than `any`, but we can save that for the PR that handles https://github.com/dcgtc/dgrants/issues/463

Reply:
Now we use type ContributionSubgraph instead of any.